lib/adal.js

```diff
@@ -870,7 +945,10 @@ var AuthenticationContext = (function () {
         return extractedUri;
     };
 
-    /*exported  oauth2Callback */
+    /**
+     * This method must be called for processing the response received from AAD. It extracts the hash, processes the token or error, saves it in the cache and calls the registered callbacks with the result.
+     * @ignore
```

Comment:
Re-commenting:
public method. This is the main method for processing responses received from AAD for users using plain JS. This MUST be called in order to process the response. This can say something like:
This method must be called for processing the response received from AAD. It extracts the hash, process the token or error, saves it in the cache and calls the registered callbacks with the result.